README.md

```diff
@@ -1 +1,12 @@
 # InnoSCS
+## What is it
+An adaptive web tool for seamless customer support across platforms by a volunteer team. This tool can be integrated into mobile and web apps, connect to Telegram through a telegram bot, and keep track of the conversation threads via a database. The main idea is to make seamless customer support which will be smooth and easily integrated.
+
+## Installation
+Open corresponding README files for frontend and backend
+
+
+## Licence
+MIT License
+
+Copyright (c) 2022 InnoSCS
```

Comment:
1. It is not possible to have a copyright and a MIT license at the same time. They are directly contradicting. MIT license allows users to copy the software freely; copyright gives the exclusive rights to do that to the owner/creator.
2. Was it a course instructor who suggested you to add this license? Because, _as far as I'm aware_, we as Innopolis students are not allowed to publish software created during the courses under this license (of course, unless you have a written permission from whichever department is responsible for that), according to our contracts. So, I think it is worth asking a course instructor (or maybe some legal guy) about that explicitly.
3. It is common to create the `LICENSE` file in the root directory of the repository (GitHub will help you to create it if you try to create the file with that name via its web interface)